resources/install.rb

```diff
@@ -175,7 +175,7 @@ def hab_path
   end
 
   def hab_command
-    cmd = if node['kernel']['release'].to_f < 3.0
+    cmd = if node['kernel']['release'].to_i < 3.0
```

Comment:
```suggestion
    cmd = if node['kernel']['release'].to_i < 3
```

Reply:
I can't believe I did that on both of those. They are fixes. Thanks pointing it out.